src/System Application/App/Edit in Excel/src/EditInExcel.Codeunit.al

```diff
@@ -14,6 +15,42 @@ codeunit 1481 "Edit in Excel"
 {
     Access = Public;
 
+    procedure CreateMetadataWebRequest()
```

Comment:
Whats with this procedure?
Does nothing? Init local variable and one declared variable not used.